Makefile

```diff
@@ -179,3 +179,11 @@ ifndef GITHUB_AUTH_TOKEN
 	$(error GITHUB_AUTH_TOKEN is undefined)
 endif
 ###############################################################################
+
+
+###############################################################################
+
+.PHONY: ko-local
+ko-local:
```

Comment:
I don't fully understand `ko` so I have a bunch of questions/comments:

1. Where is `KO_PREFIX` defined? 
2. Should it be `$()` instead of `${}`?
3. Its missing flags defined by `make build-scorecard` like version flags - https://github.com/ossf/scorecard/blob/main/Makefile#L108
4. How does this impact CloudBuild? And should we remove the existing `Dockerfile` for `scorecard`?
5. How will `ko` handle more complex Dockerfile usecases like copying over data file (like in cron/controller)?

IMO, it'll be useful to discuss the full plan in #744 before proceeding with the PR.

Reply:
> I don't fully understand `ko` so I have a bunch of questions/comments:
> 
> 1. Where is `KO_PREFIX` defined?
Ko_PREFIX would be defined the github actions before pushing it.
> 2. Should it be `$()` instead of `${}`?

> 3. Its missing flags defined by `make build-scorecard` like version flags - https://github.com/ossf/scorecard/blob/main/Makefile#L108
This is initial for pushing to `ko-local` which means this is not for pushing to gcr.io yet. It is pushing to local docker.
> 4. How does this impact CloudBuild? And should we remove the existing `Dockerfile` for `scorecard`?
In following PR's I will include things to push to gcr.io and then the cloudbuild and the existing docker file will be removed.
> 5. How will `ko` handle more complex Dockerfile usecases like copying over data file (like in cron/controller)?
It will be able to handle copying over data https://github.com/google/ko#static-assets
> 
> IMO, it'll be useful to discuss the full plan in #744 before proceeding with the PR.

Reply:
I see, in that case can we either extend this PR to provide more e2e/wholistic usage of `ko` or we can make a design before starting with a series of PRs. Currently, this PR is adding dead code which will not be touched without follow up PRs, so I would prefer not submitting it.